<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>1200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QStackedWidget" name="stackedPages">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="carInformationPage">
       <widget class="QPushButton" name="nextPageButton">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>930</y>
          <width>400</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Next Page</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="batteryPercentageBar">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>800</y>
          <width>600</width>
          <height>71</height>
         </rect>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
       <widget class="QLabel" name="pageTitleLabel">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>50</y>
          <width>200</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <underline>true</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">text-decoration: underline;</string>
        </property>
        <property name="text">
         <string>Car Information</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="imageLabel">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>150</y>
          <width>501</width>
          <height>551</height>
         </rect>
        </property>
        <property name="text">
         <string>Place Image Here</string>
        </property>
       </widget>
       <widget class="QLabel" name="batteryPercentageLabel">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>730</y>
          <width>451</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>17</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Battery Percentage</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QFrame" name="infoFrame">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>541</width>
          <height>571</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="ownerLabel">
           <property name="font">
            <font>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Owner:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="carTypeLabel">
           <property name="font">
            <font>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Car Type:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="carNameLabel">
           <property name="font">
            <font>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Car Name:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="colorLabel">
           <property name="font">
            <font>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Color:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="locationLabel">
           <property name="font">
            <font>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Location:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="memberPage">
       <widget class="QLabel" name="listNameLabel">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>20</y>
          <width>281</width>
          <height>51</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>80</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>ViscommTeam Members</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidget">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>100</y>
          <width>800</width>
          <height>591</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>800</width>
          <height>800</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QPushButton" name="addMemberButton">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>750</y>
          <width>380</width>
          <height>70</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>70</height>
         </size>
        </property>
        <property name="text">
         <string>Add Member</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pageBackwardsButton">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>910</y>
          <width>500</width>
          <height>70</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>70</height>
         </size>
        </property>
        <property name="text">
         <string>Return to Previous Page</string>
        </property>
       </widget>
       <widget class="QPushButton" name="removeMemberButton">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>750</y>
          <width>380</width>
          <height>70</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>70</height>
         </size>
        </property>
        <property name="text">
         <string>Remove Member</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="newUserPage">
       <widget class="QLabel" name="newUserTitle">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>30</y>
          <width>171</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <underline>true</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">text-decoration: underline;</string>
        </property>
        <property name="text">
         <string>New User Form</string>
        </property>
       </widget>
       <widget class="QLabel" name="inputFirstNameLabel">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>160</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>First Name:</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="firstNameTextArea">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>160</y>
          <width>251</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="inputLastNameLabel">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>310</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Last Name:</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="lastNameTextArea">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>310</y>
          <width>251</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="inputGradYearLabel">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>480</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Graduation Year:</string>
        </property>
       </widget>
       <widget class="QListWidget" name="gradYearList">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>470</y>
          <width>256</width>
          <height>61</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>2019</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2020</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2021</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2022</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2023</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2024</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2025</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2026</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="addNoUpdateButton">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>660</y>
          <width>311</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Add Without Updating JSON File</string>
        </property>
       </widget>
       <widget class="QLabel" name="errorLabel">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>590</y>
          <width>761</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: red</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="addAndUpdateButton">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>660</y>
          <width>311</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Add and Update JSON File</string>
        </property>
       </widget>
       <widget class="QPushButton" name="noAddButton">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>800</y>
          <width>311</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="removeUserPage">
       <widget class="QLabel" name="removeFormTitle">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>80</y>
          <width>221</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <underline>true</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">text-decoration: underline;</string>
        </property>
        <property name="text">
         <string>Remove User Form</string>
        </property>
       </widget>
       <widget class="QLabel" name="removeInstructionsLabel">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>180</y>
          <width>521</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Please select the users to remove from the list below:</string>
        </property>
       </widget>
       <widget class="QListWidget" name="toRemoveList">
        <property name="geometry">
         <rect>
          <x>265</x>
          <y>280</y>
          <width>611</width>
          <height>241</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="removeNoUpdateButton">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>660</y>
          <width>311</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Remove Without Updating JSON File</string>
        </property>
       </widget>
       <widget class="QPushButton" name="removeAndUpdateButton">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>670</y>
          <width>311</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Remove and Update JSON File</string>
        </property>
       </widget>
       <widget class="QPushButton" name="noRemoveButton">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>820</y>
          <width>311</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QLabel" name="removeErrorLabel">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>560</y>
          <width>351</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: red</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
